DeAceroDLabs/marketplace-frontend
# Feature/dmp 516 Error popup in signup.

## What is done

[[DMP-516] Error popup in signup.](https://deacero.atlassian.net/browse/DMP-516)

## How is done

- Created a `signupMessages` file that will receive the error or successful messages when signing up.

## How to test

1. Run the project via `npm start`
2. signup.
3. Fill everynthing but with one email that is already in Auth0. At the end you should see the error popup.
4. Fill everynthing but with one email that is not in Auth0. At the end you should see the success popup.
